src/code42cli/cmds/auditlogs.py

```diff
@@ -0,0 +1,255 @@
+import json
+from datetime import datetime
+from datetime import timezone
+from datetime import timedelta
+from _collections import OrderedDict
+
+import click
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.cmds.search.cursor_store import AuditLogCursorStore
+from code42cli.cmds.search.options import BeginOption
+from code42cli.date_helper import parse_max_timestamp
+from code42cli.date_helper import parse_min_timestamp
+from code42cli.logger import get_logger_for_server
+from code42cli.options import begin_option
+from code42cli.options import end_option
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.options import send_to_format_options
+from code42cli.options import server_options
+from code42cli.output_formats import OutputFormatter
+from code42cli.util import warn_interrupt
+
+EVENT_KEY = "events"
+AUDIT_LOGS_KEYWORD = "audit-logs"
+AUDIT_LOG_TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
+
+AUDIT_LOGS_DEFAULT_HEADER = OrderedDict()
+AUDIT_LOGS_DEFAULT_HEADER["timestamp"] = "Timestamp"
+AUDIT_LOGS_DEFAULT_HEADER["type$"] = "Type"
+AUDIT_LOGS_DEFAULT_HEADER["actorName"] = "ActorName"
+AUDIT_LOGS_DEFAULT_HEADER["actorIpAddress"] = "ActorIpAddress"
+AUDIT_LOGS_DEFAULT_HEADER["userName"] = "AffectedUser"
+AUDIT_LOGS_DEFAULT_HEADER["userId"] = "AffectedUserUID"
+# AUDIT_LOGS_DEFAULT_HEADER["success"] = "Success"
+# AUDIT_LOGS_DEFAULT_HEADER["resultCount"] = "ResultCount"
+
+filter_option_usernames = click.option(
+    "--username", required=False, help="Filter results by usernames.", multiple=True,
+)
+filter_option_user_ids = click.option(
+    "--user-id", required=False, help="Filter results by user ids.", multiple=True,
+)
+
+filter_option_user_ip_addresses = click.option(
+    "--user-ip",
+    required=False,
+    help="Filter results by user ip addresses.",
+    multiple=True,
+)
+filter_option_affected_user_ids = click.option(
+    "--affected-user-id",
+    required=False,
+    help="Filter results by affected user ids.",
+    multiple=True,
+)
+filter_option_affected_usernames = click.option(
+    "--affected-username",
+    required=False,
+    help="Filter results by affected usernames.",
+    multiple=True,
+)
+filter_option_event_types = click.option(
+    "--event-type",
```

Comment:
When searching file events, `-t` represent `Exposure Type`... Which, now in retrospect is confusing.  Users may not know what they are searching for, especially when you limit the logs by `Event Type` rather. Does this add even more to that uncertainty?

If there is a 2.0 list for the CLI, I propose we rename `-t` for file events searches.